### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/customize/layout.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#846] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/customize/layout.html

此教程指导你如何修改主题布局。

https://theme-hope.vuejs.press/zh/guide/customize/layout.html

**Comment by fkomorebi** [comment]
我想对主题内默认的Layout做一些个性化的修改，我该如何使用这个Layout?我尝试过使用 ‘import Layout from 'vuepress-theme-hope/lib/client/layouts/layout';’但是结果是 Internal server error: Missing "./lib/client/layouts/layout" specifier in "vuepress-theme-hope" package。这让我有点抓狂。

**Reply by Mister-Hope** [reply]
vuepress-theme-hope/layouts/layout.js

**Comment by yangliangxii** [comment]


**Reply by Mister-Hope** [reply]
可能叫Darkmode，我回头校验一下，确认不生效的话请开个issue以免我忘了

**Reply by yangliangxii** [reply]
换成Markdown也没效果。

**Comment by yangliangxii** [comment]
navbarLayout: {
start: ["Brand"],
center: ["Links"],
end: ["Search", "Repo","LoginComponent", "DarkMode"], // 这里添加 LoginComponent
// end: "auto", 使用默认的 end，然后追加 LoginComponent
}, 在themeconfig中新添加LoginComponent，DarkMode 不生效了

**Reply by Mister-Hope** [reply]
那个东西叫Outlook，而且我在文档里找了 3 遍我没看到文档的任何部分是错误的。You don't pay me, so don't waste my time.

**Reply by yangliangxii** [reply]
呵呵，终于显示了。谢谢大佬~~~
